automation/taskcluster/decision_task.py

```diff
@@ -66,13 +65,13 @@ def pr_or_push(is_push):
     signing_tasks = {}
     other_tasks = {}
 
-    for variant in debug_variants:
+    for variant in get_debug_variants():
         assemble_task_id = taskcluster.slugId()
         build_tasks[assemble_task_id] = BUILDER.craft_assemble_task(variant)
         build_tasks[taskcluster.slugId()] = BUILDER.craft_test_task(variant)
 
     if is_push and SHORT_HEAD_BRANCH == 'master':
-        for variant in ('armRaptor', 'aarch64Raptor'):
+        for variant in [Variant.from_values(abi, False, 'raptor') for abi in ('aarch64', 'arm')]:
```

Comment:
I particularly like this. I presume if we add a new platform in the future, it'll be automatically provided by the gradle call, rather than enumerate it here 👍

Reply:
Note that in this case, we'll still have to manually any new platforms that we want to send to raptor.
I think that this is ideal behaviour, considering that so far we've been filtering for specific platforms already.